scripts/helmAcs.sh

```diff
@@ -196,7 +196,7 @@ repository:
   livenessProbe:
     initialDelaySeconds: 420
   adminPassword: \"$ALFRESCO_PASSWORD\"
-  `if [ ! -z ${REPO_IMAGE} ] || [ ! -z ${REPO_TAG} ]; then echo image: ; fi`
+  `if [ ! -z ${REPO_IMAGE} ] || [ ! -z ${REPO_TAG} ]; then echo image: ; else echo -e 'image: \n    repository: "alfresco/alfresco-content-repository-aws"' ; fi`
```

Comment:
I don't think this logic is correct.

if someone just provides an override for "REPO_TAG" you are going to end up with:

```
image:
  tag: "<REPO_TAG>"
```

i.e. the repository will be missing again!

Maybe change it so "image:" is always output (as there will always be a repository key) and then deal with the REPO_IMAGE and REPO_TAG parameters separately.